salt/modules/keystoneng.py

```diff
@@ -0,0 +1,382 @@
+import salt.utils
```

Comment:
We need the `# -*- coding: utf-8 -*-` and then a doc string containing the information about how to setup the authentication for this module.